validator/rpc/standard_api.go

```diff
@@ -133,14 +164,6 @@ func (s *Server) DeleteKeystores(
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not delete keys: %v", err)
 	}
-	if len(statuses) != len(req.PublicKeys) {
-		return nil, status.Errorf(
-			codes.Internal,
-			"Wanted same amount of statuses %d as public keys %d",
-			len(statuses),
-			len(req.PublicKeys),
-		)
-	}
```

Comment:
Why remove this one?

Reply:
I didn't think the check was required, the function always returns the same length as the keys sent in from how I read it. please let me know if I'm wrong on that